e2e/cypress/integration/channel/scroll_channel_messages_spec.js

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+import users from '../../fixtures/users';
+
+const sysadmin = users.sysadmin;
+
+describe('Scroll channel`s messages in mobile view', () => {
+    let newChannel;
+
+    before(() => {
+        cy.apiLogin('user-1');
+
+        // # resize browser to phone view
+        cy.viewport('iphone-6');
+
+        // # visit channel
+        cy.createAndVisitNewChannel().then((channel) => {
+            newChannel = channel;
+        });
+    });
+
+    it('M18759 - detect change in floating timestamp', () => {
+        let date;
+
+        // # Post a day old message
+        for (let j = 2; j >= 0; j--) {
+            date = Cypress.moment().subtract(-j, 'days').valueOf();
+            for (let i = 0; i < 5; i++) {
+                cy.postMessageAs({sender: sysadmin, message: `Hello \n from \n other \n day \n - ${j}`, channelId: newChannel.id, createAt: date});
+            }
+        }
+
+        // # reload to see correct changes
+        cy.reload();
+
+        const yesterdaysDate = Cypress.moment().subtract(-1, 'days');
+        const twoDaysAgo = Cypress.moment().subtract(0, 'days');
+
+        // # set date 3 days fron now because channel created today and scroll not working because of it
+        cy.clock(Cypress.moment().subtract(-3, 'days').valueOf(), ['Date']);
```

Comment:
I suggest to add only 2 days from now so it could have test for "Today" floating timestamp.
```suggestion
        cy.clock(Cypress.moment().add(2, 'days').valueOf(), ['Date']);
```